### ISSUE TITLE
Use parallel computing while parsing

### ISSUE THREAD

**Issue by kulla** [Lodifice/mfnf-pdf-export#83]

**Comment by vroland** [comment]
Actually, the problem here is API call caching. To properly implement this, we need a caching database backend which supports concurrency. Does anyone know a simple solution for this?

**Comment by kulla** [comment]
@Lodifice Have you an idea?!

**Comment by kulla** [comment]
Same opinion here :-) We should work on this issue after we have implemented everything else for milestone 1.0.

**Comment by vroland** [comment]
Makes sense. I tried parallel parsing of articles with individual caches for each article, but this yields way worse execution times.
